### DISCUSSION TITLE
Maybe consider binding a shortcut key to the [Override - I was correct] button

### DISCUSSION THREAD

**Discussion by shuiRong** [quenti-io/quenti#327] [Ideas]
<img width="879" alt="image" src="https://github.com/quenti-io/quenti/assets/16076993/0182b79b-d344-406f-aa93-ece6345dbeb7">

I found that Quenti (Quizlet) checks both phonetics and characters when verifying the correctness of Japanese input, it is quite cumbersome. If I can type the characters, there is no need to also require phonetics at the same time. So, I have to manually click the "override" button each time, and it would be great if there were a shortcut key for this. Then, I wouldn't need the mouse at all. Please consider adding this feature, and I appreciate it very much.

Additionally, I was thinking about implementing this feature myself as a web frontend developer. However, it seems that these three modules are not open source, so I cannot run the project locally.

Thank you once again for your project.

**Comment by miapolis** [comment]
Thank you for bringing this up! Yeah, the way all answer evaluation works is based off of characters. There's some special rules depending on the language for not ignoring parentheses when answering in Math for example, but that's about it. 

All evaluation code is handled [here](https://github.com/quenti-io/quenti/blob/85d0fbe912da7f151354884d1c0e20aa2f20952b/packages/core/evaluator.ts) and if you're looking for the write mode specifics that's in [this directory](https://github.com/quenti-io/quenti/tree/85d0fbe912da7f151354884d1c0e20aa2f20952b/apps/next/src/modules/learn/cards/write).

In terms of the not open source modules, I've just update the package.json so bun doesn't throw an error anymore. Try running `bun i` one more time.
